web/main/views.py

```diff
@@ -568,8 +566,10 @@ class SectionTOCView(APIView):
     @method_decorator(user_has_perm("casebook", "viewable_by"))
     @method_decorator(user_has_perm("section", "viewable_by"))
     def get(self, request, casebook, section, format=None):
-        section.content_tree__load()
-        return Response(SectionOutlineSerializer(section).data)
+        mscq = manually_serialize_content_query(
+            ContentNode.objects.filter(id=section.id) | section.contents
+        )[0]
```

Comment:
I have a syntax recommendation! To make it clear that you expect `manually_serialize_content_query(...)` to return a list of length one and extract that item at the same time, you can:
```
[mscq] = manually_serialize_content_query(....)
```
I think that would be a bit clearer than `[0]`, which could make a person wonder why we aren't returning the [non-existent) rest of the list too.